src/torchmetrics/image/inception.py

```diff
@@ -96,6 +98,7 @@ def __init__(
         self,
         feature: Union[str, int, Module] = "logits_unbiased",
         splits: int = 10,
+        normalize: bool = False,
```

Comment:
```suggestion
        normalized: bool = False,
```
`normalize` sounds to me as a normalize will be performed on the input in metric...

Reply:
Just trying to keep it consistent with the other image metric which also has this kind of argument:
https://github.com/Lightning-AI/metrics/blob/169ec1e4e1773f4c993ef27356bbf06d4cab08da/src/torchmetrics/image/lpip.py#L101

Reply:
I see, so in such a case would it make sense to have another PR with changing all args?